apps/pyth-oracle/README.md

```diff
@@ -0,0 +1,105 @@
+# Query Berachain Data with Pyth Oracles
+
+[Pyth Network](https://pyth.network/) is a decentralized oracle network that provides high-quality, real-time data for Berachain. This repo provides an example of how to query Pyth Network data on Berachain's Artio Testnet.
+
+Pyth differs from the existing oracle paradigm by using on-demand price updates, where users pull on-chain prices only when needed - learn more [here](https://pyth.network/blog/pyth-a-new-model-to-the-price-oracle).
+
+## Requirements
+
+- Nodejs `v20.11.0` or greater
+- pnpm (or another prefered package manager)
+- [jq ](https://jqlang.github.io/jq/download/)
+- Wallet with testnet $BERA tokens - See the [Berachain Artio Faucet](https://artio.faucet.berachain.com)
+- [Foundry](https://book.getfoundry.sh/getting-started/installation) - ensure `foundryup` is run to install binaries
+
+## Quick Setup
+
+### Step 1 - Setup Project & Install Dependencies
+
+```bash
+# FROM: ./pyth-oracle
+
+pnpm install;
+```
+
+Add remappings to `./foundry.toml` with the following:
+
+```toml
+echo "remappings = ['@pythnetwork/pyth-sdk-solidity/=node_modules/@pythnetwork/pyth-sdk-solidity']" >> ./foundry.toml
+```
```

Comment:
Might not need, because already in your foundry.toml, but good for your tutorial to mention to configure this.